### PR TITLE
TELCODOCS-874: New feature release notes for Metal platform for 4.12

### PR DESCRIPTION
**Pull request by rohennes**
[TELCODOCS-874](https://issues.redhat.com//browse/TELCODOCS-874): New feature release notes for Metal platform for 4.12

Version(s):
4.12

Issue:
https://issues.redhat.com/browse/TELCODOCS-874

Link to docs preview:
http://file.emea.redhat.com/rohennes/TELCODOCS-874-RNs-4.12/release_notes/ocp-4-12-release-notes.html

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -92,6 +92,16 @@ For more information about enabling a feature set during installation, see xref:
...
 
+[id="ocp-4-12-consistent-address-for-ironicapi"]
+==== Consistent IP address for Ironic API in bare-metal installations without a provisioning network  
+With this update, in bare-metal installations without a provisioning network, the Ironic API service is accessible through a proxy server. This proxy server provides a consistent IP address for the Ironic API service. If the Metal3 pod that contains `metal3-ironic` relocates to another pod, the consistent proxy address ensures constant communication with the Ironic API service.
+
+[id="ocp-4-12-ironic-container-image-rhel9"]
+==== Ironic container images use RHEL 9 base image
+In earlier versions of {product-title}, Ironic container images used Red Hat Enterprise Linux (RHEL) 8 as the base image. From {product-title} 4.12, Ironic container images use RHEL 9 as the base image. The RHEL 9 base image adds support for CentOS Stream 9, Python 3.8, and Python 3.9 in Ironic components.
```
Comment (skrthomas) [inline release_notes/ocp-4-12-release-notes.adoc:101]:
```suggestion
In earlier versions of {product-title}, Ironic container images used Red Hat Enterprise Linux (RHEL) 8 as the base image. Starting with {product-title} 4.12, Ironic container images use RHEL 9 as the base image. The RHEL 9 base image adds support for CentOS Stream 9, Python 3.8, and Python 3.9 in Ironic components.
```

Comment (skrthomas) [reply release_notes/ocp-4-12-release-notes.adoc:101]: Since 4.12 is the new release I think this seems more fitting verbiage. Small nit.

**Review by skrthomas — COMMENTED** [review]
A small nit otherwise lgtm

**Comment by michaelryanpeter** [comment]
@rohennes It looks like you are having build problems due to an `xref` [here](https://github.com/openshift/openshift-docs/pull/53814/files#diff-b23359d6c6c72e1685722b131e03ac147e26629c96452ccfec5960b3274b0873R570). Would you please take a look?

**Comment by rohennes** [comment]
> @rohennes It looks like you are having build problems due to an `xref` [here](https://github.com/openshift/openshift-docs/pull/53814/files#diff-b23359d6c6c72e1685722b131e03ac147e26629c96452ccfec5960b3274b0873R570). Would you please take a look?

Thanks @michaelryanpeter - that ZTP section was revamped so my link was out of date. I updated the link accordingly.

**Comment by rohennes** [comment]
Merged these separately as some EPICs related to RNs were not in a release_pending state. Closing this PR.

**Pull request closed without merging**